discovery/azure/azure.go

```diff
@@ -170,13 +172,32 @@ func createAzureClient(cfg SDConfig) (azureClient, error) {
 	resourceManagerEndpoint := env.ResourceManagerEndpoint
 
 	var c azureClient
+
 	oauthConfig, err := adal.NewOAuthConfig(activeDirectoryEndpoint, cfg.TenantID)
```

Comment:
shouldn't it be moved to the "OAuth" switch case too?

Reply:
You're right, just changed it.